### PR TITLE
PXB-2672 Create 8.0.27-19 Release notes and Release Note tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
M  source/conf.py
M  source/release-notes.rst
A  source/release-notes/8.0/8.0.27-19.0.rst

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* :jirabug:`PXB-2434`: Use MySQL page tracking for incremental backup
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-1741`: PS startup displays errors for databases excluded during partial backup
+* :jirabug:`PXB-2614`: The ``xbstream`` binary sparse files now supports the XFS file system.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst]: files now **support**

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, it drives down backup costs while providing unique features for MySQL backups.
+
+Release Highlights
+=================================================
+
+The following list are some of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated.  Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst]: is deprecated.  Support ->  remove the extra space

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+:Date: January  2022
+:Installation: `Installing Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html>`_
+
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, it drives down backup costs while providing unique features for MySQL backups.
+
+Release Highlights
+=================================================
+
+The following list are some of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
```
Comment (alina-derkach-oaza) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst]: The following list **are** -> The following list **contains**?

Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst]: Rewritten.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst (hunk 2 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst]: Removed.

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst]: Rewritten.

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by george-lorch — APPROVED** [review]

**Pull request merged**